.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,35 @@
+name: Feature Request 💡
+description: Have any new idea or new feature ? Please suggest!
+title: "[Feature] <write a small description here>"
+labels: ["enhancement", "goal: addition"]
```

Comment:
Add `feature` label here. Check the `labels` in the issues section. I have created a new `feature` label.